book/src/group-snapshot.md

```diff
@@ -0,0 +1,165 @@
+
+To implement the volume group snapshot feature, a CSI driver MUST:
+
+* Implement a new group controller service.
+* Implement group controller RPCs: `CreateVolumeGroupSnapshot`, `DeleteVolumeGroupSnapshot`, and `GetVolumeGroupSnapshot`.
+* Add group controller capability `CREATE_DELETE_GET_VOLUME_GROUP_SNAPSHOT`.
+
+For details,  see the [CSI spec](https://github.com/container-storage-interface/spec/blob/master/spec.md).
+
+## Sidecar Deployment
+
+The Kubernetes CSI development team maintains the
+[external-snapshotter](external-snapshotter.md) Kubernetes CSI [Sidecar
+Containers](sidecar-containers.md). This sidecar container implements the logic
+for watching the Kubernetes API objects and issuing the appropriate CSI volume
+group snapshot calls against a CSI endpoint. For more details, see
+[external-snapshotter documentation](external-snapshotter.md).
+
+## Volume Group Snapshot APIs
+
+With the introduction of Volume Group Snapshot, the user can create and delete a
+group snapshot using Kubernetes APIs.
+
+The schema definition for the custom resources (CRs) can be found
+[here](https://github.com/kubernetes-csi/external-snapshotter/tree/master/client/config/crd).
+The CRDs should be installed by the Kubernetes distributions.
+
+There are 3 APIs:
+
+`VolumeGroupSnapshot`
+: Created by a Kubernetes user (or perhaps by your own automation) to request
+creation of a volume group snapshot for multiple volumes.
+It contains information about the volume group snapshot operation such as the
+timestamp when the volume group snapshot was taken and whether it is ready to use.
+The creation and deletion of this object represents a desire to create or delete a
+cluster resource (a group snapshot).
+
+`VolumeGroupSnapshotContent`
+: Created by the snapshot controller for a dynamically created VolumeGroupSnapshot.
+It contains information about the volume group snapshot including the volume group
+snapshot ID.
+This object represents a provisioned resource on the cluster (a group snapshot).
+The VolumeGroupSnapshotContent object binds to the VolumeGroupSnapshot for which it
+was created with a one-to-one mapping.
+
+`VolumeGroupSnapshotClass`
+: Created by cluster administrators to describe how volume group snapshots should be
+created. including the driver information, the deletion policy, etc.
+
+## Controller
+
+* The controller logic for volume group snapshot is added to the snapshot
+  controller and the CSI external-snapshotter sidecar.
+
+The snapshot controller is deployed by the Kubernetes distributions and is
+responsible for watching the VolumeGroupSnapshot CRD objects and manges the
+creation and deletion lifecycle of volume group snapshots.
+
+The CSI external-snapshotter sidecar watches Kubernetes
+VolumeGroupSnapshotContent CRD objects and triggers
+CreateVolumeGroupSnapshot/DeleteVolumeGroupSnapshot against a CSI endpoint.
+
+## Snapshot Validation Webhook
+
+The validating webhook server is updated to validate volume group snapshot
```

Comment:
Can you add a note similar to what we have in the release note that the webhook is deprecated?
https://github.com/kubernetes-csi/external-snapshotter/releases/tag/client%2Fv8.0.0